.scripts/process_build.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
+
+# This script is called from everest-ci/ci script for a weekly build of the FStar Binaries
+# If ran separately, the starting directory should be the root directory of FStar.
+
+# Sorry, everyone
+if (( ${BASH_VERSION%%.*} < 4 )); then
+  echo "This script requires Bash >= 4. On OSX, try: brew install bash"
+  exit 1
+fi
+
+# Any error is fatal.
+set -e
+set -o pipefail
+
```

Comment:
Nit: can you put the definition of ORIG_PWD right here, before it's first used? It would make it easier to follow where the script starts from (the root of the git "binaries" repository).